### ISSUE TITLE
VS2017 RC regression - Namespace completions are no longer filtered

### ISSUE THREAD

**Issue by dsyme** [Microsoft/visualfsharp#1805]
In VS2012-VS2015, completions at ``open`` declarations were filtered, so pressing ``.`` here would only show the namespaces and modules that are valid completions, 

    open System.Collections.

In VS2017 RC, all types are also displayed.

#### Repro steps

Provide the steps required to reproduce the problem

1. Create an F# Project

2. Try a completion of the text `` open System.Collections.``

#### Expected behavior

A completion list showing only namespaces

#### Actual behavior

A completion list showing namespaces and types

```
Microsoft Visual Studio Professional 2017 RC
Version 15.0.25920.0 D15REL
Microsoft .NET Framework
Version 4.6.01038

Installed Version: Professional

Visual Basic 2017 RC   00369-50000-00000-AA495
Microsoft Visual Basic 2017 RC

Visual C# 2017 RC   00369-50000-00000-AA495
Microsoft Visual C# 2017 RC

Visual F# 4.1   00369-50000-00000-AA495
Microsoft Visual F# 4.1

Application Insights Tools for Visual Studio Package   8.3.01031.3
Application Insights Tools for Visual Studio

ASP.NET and Web Tools 2016   15.0.21108.0
ASP.NET and Web Tools 2016

ASP.NET Web Frameworks and Tools 2013   5.2.41027.0
For additional information, visit https://www.asp.net/

Azure App Service Tools v2.9.6   15.0.21107.0
Azure App Service Tools v2.9.6

Common Azure Tools   1.8
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

JavaScript Language Service   2.0
JavaScript Language Service

NuGet Package Manager   4.0.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

SQL Server Data Tools   15.1.61101.0
Microsoft SQL Server Data Tools

TypeScript   2.1.2.0
TypeScript tools for Visual Studio
```

**Comment by dsyme** [comment]
@OmarTawfik Any ideas?  This was covered by old Language Service unit tests.

**Comment by dsyme** [comment]
@OmarTawfik @brettfo I've found the root cause for this bug and will send a PR soon

**Comment by dsyme** [comment]
Fixed by #1816
